package/package.json

```diff
@@ -2,23 +2,13 @@
   "name": "@forgerock/login-widget",
   "type": "module",
   "exports": {
-    "inline": {
-      "import": "./inline.js",
-      "require": "./inline.cjs"
-    },
-    "modal": {
-      "import": "./modal.js",
-      "require": "./modal.cjs"
-    }
+    "import": "./index.js",
+    "require": "./index.cjs"
   },
+  "types": "./index.d.ts",
   "typesVersions": {
     "*": {
-      "modal": [
-        "./modal.d.ts"
-      ],
-      "inline": [
-        "./inline.d.ts"
-      ]
+      "index": ["./index.d.ts"]
```

Comment:
I'm questioning if we need this `typesVersions`. This seems to be for supporting versions of TypeScript, which I don't think we need to do. If we use the `types` key, I think that's all we need.

Reply:
in the case of one entry file, we can remove this yes.
this was a hack i used to get the two type definitions to work